api/api_public_blockchain.go

```diff
@@ -649,6 +657,16 @@ func newRPCTransactionFromBlockHash(b *types.Block, hash common.Hash) map[string
 }
 
 func (args *CallArgs) ToMessage(globalGasCap uint64, baseFee *big.Int, intrinsicGas uint64) (*types.Transaction, error) {
+	if args.GasPrice != nil && (args.MaxFeePerGas != nil || args.MaxPriorityFeePerGas != nil) {
+		return nil, errors.New("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified")
+	} else if args.GasPrice == nil && args.MaxFeePerGas == nil {
+		return nil, errors.New("the one of gasPrice and maxFeePerGas is not specified")
```

Comment:
if-condition says that both of `args.GasPrise` and `args.MaxFeePerGas` are not specified.